src/features/wallet/root/ActivityCard/ActivityCardHeader.tsx

```diff
@@ -60,4 +68,4 @@ const ActivityCardHeader = ({ filter, onFilterChanged }: Props) => {
   )
 }
 
-export default memo(ActivityCardHeader)
```

Comment:
what's the purpose of putting memo on every export? should we be doing this everywhere? is it necessary?